src/main/java/core/basesyntax/model/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.model;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started.");
```

Comment:
The message in the `doWork` method should be more informative. It should indicate that the type of `Machine` working right now is `Truck`. Consider rephrasing it to something like 'Truck is doing its work.'